student.rb

```diff
@@ -0,0 +1,120 @@
+require 'csv'
+require_relative 'calculator'
+class Student
+
+	attr_accessor :name, :roll_number, :bengali, :english, :maths, :computer, :total, :percentage, :percentile
+	def initialize name, roll, bengali,english,maths,computer,t,per
```

Comment:
set default value for total and percentage

Reply:
both are already initialized, should keep them as 0?